pkg/client/package_extension.go

```diff
@@ -0,0 +1,61 @@
+package client
```

Comment:
Is it possible to add a test for this file? (Like we have for package_buildpack.go)

Reply:
Yes, I'm actively working on it, will be finished